### ISSUE TITLE
make index html beautiful again

### ISSUE THREAD

**Issue by FelixTe** [ReTtiChDieOrganisation/ReTtiChDasRepo#50]
Ich musste gerade sehr lange nach der draw segment funktion suchen. Vielleicht können wir das alles so umsortieren, dass die funktionen alle zusammen weiter oben sind. Weiß aber nicht wie man das style mäßig am besten macht

**Comment by prosauer** [comment]
ja, glaube dass wir bisschen refactoren sollten. Also auch Trennung zwischen dem CSS-Zeug und den HTML/JS Sachen
